capstone/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -15,6 +20,26 @@
 @WebServlet("/form-handler")
 public class FormHandlerServlet extends HttpServlet {
 
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        List<Form> forms = new ArrayList<>();
```

Comment:
Declare the list right next to where it will be used, so right above the for loop

Reply:
Got it. Most of the time, I would create any new objects in mind at the top but now, I'll make it so that it'll be created when relevant.